File: autocrm/collectors/beeper.py
"""Beeper Desktop collector → outbox."""

from autocrm.collectors.collector import Collector, CollectResult


class BeeperCollector(Collector):
    app = "beeper"

    def collect(self) -> CollectResult:
        # TODO: call Beeper Desktop local API, enqueue events, advance cursor.
        return CollectResult(source=self.app, enqueued=0)

File: internal/collectors/beeper.go
package collectors

// BeeperCollector reads Beeper Desktop into the outbox (stub).
type BeeperCollector struct{}

func (BeeperCollector) App() string { return "beeper" }

func (BeeperCollector) Collect() (CollectResult, error) {
	return CollectResult{Source: "beeper", Enqueued: 0}, nil
}
